Reflection/ReflectionAttribute.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @since 8.0
+ */
+class ReflectionAttribute
```

Comment:
Please correct me if I'm wrong but seems several methods are missed in this class . For example https://github.com/php/php-src/blob/master/ext/reflection/php_reflection.c#L6460 . Or please confirm if it was intended to skip those methods for now

Reply:
This RFC was merged recently: https://wiki.php.net/rfc/attribute_amendments

At the time of preparing this PR, these methods were not yet in the master branch. So yes, several methods have been skipped. Some I added marked "TODO" comment.

Since they are already merged, it makes sense to add them already.

Reply:
Added https://github.com/JetBrains/phpstorm-stubs/pull/857/commits/bda34ef1fd712f5f8bf2f3cde0eb29889aaa1d5a